### PR TITLE
Added a mutli-stage Dockerfile

### PR DESCRIPTION
**Pull request by chriscowley**

### CODE REVIEW

**Comment by SuperQ** [comment]
How about a lighter-weight base image, like `quay.io/prometheus/busybox`.

See the [Prometheus Dockerfile](https://github.com/prometheus/prometheus/blob/master/Dockerfile)

**Comment by chriscowley** [comment]
Not a bad idea - gains 2MB :)

File: README.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ To see all available configuration flags:
 
     ./pgbouncer_exporter -h
 
+## Run with Docker
+
+```
+docker run chriscowley/pgbouncer_exporter -pgBouncer.connectionString=postgres://postgres:@pgbouncer:6432/pgbouncer?sslmode=disable
```
Comment (stanhu) [inline README.md]: Do we need to update this line?

Comment (chriscowley) [reply README.md]: It is dependent on how the user's environment, so potentially yes

I have reverted back to using Alpine as the final base image. Building with Alpine (and thus musl) then runnning on busybox (ulibc iirc) was not working.

**Comment by brennoo** [comment]
Hey could we resume this ?  It would be nice to have public docker images following the releases

**Comment by SuperQ** [comment]
I've been thinking about what to do about this fork. I'd like there to be one official pgbouncer exporter. I was hoping to get some comments in https://github.com/larseen/pgbouncer_exporter/issues/13 about either resuming upstream work or hard-forking things over to prometheus-community.

**Comment by SuperQ** [comment]
FYI, I've filed https://github.com/prometheus-community/community/issues/21. I'm intending to do a hard fork, so that the code here would become the canonical source.

**Comment by SuperQ** [comment]
We've completed the move to prometheus-community. I've added our standard multi-arch Dockerfile.

It's now available on [prometheuscommunity/pgbouncer-exporter](https://hub.docker.com/repository/docker/prometheuscommunity/pgbouncer-exporter).

**Pull request closed without merging**